distributed/scheduler.py

```diff
@@ -4893,49 +4893,39 @@ async def remove_worker_from_events():
 
         return "OK"
 
-    async def stimulus_cancel(self, keys, client, force=False):
+    def stimulus_cancel(
+        self, keys: Sequence[str], client: str, force: bool = False
+    ) -> None:
         """Stop execution on a list of keys"""
         logger.info("Client %s requests to cancel %d keys", client, len(keys))
         if client:
             self.log_event(
                 client, {"action": "cancel", "count": len(keys), "force": force}
             )
-
-        await asyncio.gather(
-            *[self._cancel_key(key, client, force=force) for key in keys]
-        )
-
-    async def _cancel_key(self, key, client, force=False):
-        """Cancel a particular key and all dependents"""
-        # TODO: this should be converted to use the transition mechanism
-        ts = self.tasks.get(key)
-        try:
-            cs = self.clients[client]
-        except KeyError:
-            return
-
-        # no key yet, lets try again in a moment
-        start = time()
-        while ts is None or not ts.who_wants:
-            await asyncio.sleep(0.1)
-            ts = self.tasks.get(key)
-            if time() - start >= 1:
+        report = list()
+        clients = list()
```

Comment:
nit:
```suggestion
        report = []
        clients = []
```

Reply:
nit: Maybe `keys` or `cancelled_keys` would be a more suitable name for `report`?